pkg/rpcclient/rpc.go

```diff
@@ -151,12 +151,23 @@ func (c *Client) GetBlockHash(index uint32) (util.Uint256, error) {
 	return resp, nil
 }
 
-// GetBlockHeader returns the corresponding block header information from a serialized hex string
-// according to the specified script hash. In-header stateroot option must be
-// initialized with Init before calling this method.
+// GetBlockHeader returns the corresponding block header information from
+// a serialized hex string according to the specified script hash. In-header
+// stateroot option must be initialized with Init before calling this method.
 func (c *Client) GetBlockHeader(hash util.Uint256) (*block.Header, error) {
```

Comment:
@roman-khimov to make it compatible with `GetBlockByHash`/`GetBlockByIndex` pair we need to rename this method to `GetBlockHeaderByHash`, but it's a breaking change. I'd say we can afford this, it's not the most popular API and I'm sure that our users will handle this substitution. Are we OK with it?